xapian-applications/omega/templates/query

```diff
@@ -25,7 +25,7 @@ function write_date(time_t) {
 var d=new Date();d.setTime(time_t*1000);
 var D=d.getFullYear()+"-"+f2(d.getMonth()+1)+"-"+f2(d.getDate());
 var T=f2(d.getHours())+":"+f2(d.getMinutes())+":"+f2(d.getSeconds());
-document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
+document.write("<time datetime=\""+D+" "+T+"\" title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></time>");
```

Comment:
From my previous comment:

> so we will probably end up with both — title and element contents that our Javascript will localised, and datetime which will have the full timezone, probably in UTC for simplicity

But that isn't the case here; `datetime` and `title` have exactly the same content. Although I think this is technically a valid use of datetime, it's  ambiguous without the timezone. Likewise, it'd be good to put the timezone in the static version below (which will be in UTC, because that's how `$date{}` works.)

Reply:
Thank you for pointing this out, I have reworked the code and have added the Zulu time zone (Z) to the 'datetime' attribute.
 Please let me know if this is correct or should I make some changes.

Reply:
This looks good to me at a quick glance. Note that in future it's a good idea to use fixup commits rather than creating completely new ones. The total change here is going to end up as a single commit (because the noscript version is technically wrong without the second commit), and fixup commits make it easier to squash things together later. Not a problem with only a couple of commits, but when you're dealing with larger PRs with half a dozen commits, using fixups to indicate which commit you're updating can save a lot of time.

Reply:
Thank you for letting me know, I have squashed the commits and have updated the PR with a single commit. From now on, I will be using fixup commits for the same.